deps/build.jl

```diff
@@ -63,14 +89,15 @@ finufftbuild =
             ChangeDirectory(rootdir)
             buildcmd
             CreateDirectory(libdir(libfinufft))
-            `cp $buildfile $libfile`
+            # `cp $buildfile $libfile`
         end
     end
 run(finufftbuild)
+cp(buildfile, libfile, force=true)
```

Comment:
The change is needed on windows - otherwise there is an IOError, i.e., the cp command probably has to be used differently. Instead of adding another OS check, I thought I'd rather use the copy command provided by julia.

Reply:
Ah, makes sense!